docs/usage/permissions.rst

```diff
@@ -0,0 +1,55 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2015-2019 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+
+Access control
+--------------
+
+Invenio-Files-Rest depends on `Invenio-Access
```

Comment:
-> Invenio-Files-REST to be consistent everywhere

what about `depends` -> `takes advantage of` or similar?